package.json

```diff
@@ -76,5 +79,12 @@
     ],
     "coverageDirectory": "../coverage",
     "testEnvironment": "node"
+  },
+  "lint-staged": {
+    "*": "prettier --write --ignore-unknown",
```

Comment:
not here. add that flag to `pre-commit` file.

Reply:
it works even if we add it here

Reply:
doesn't work on my system tho.